model/qti/Element.php

```diff
@@ -722,7 +726,7 @@ protected function getArraySerializedElementCollection(
         array $elements,
         bool $filterVariableContent = false,
         array &$filtered = []
-    ): array {
+    ) {
         if (empty($elements)) {
             $data = new stdClass();
```

Comment:
I think it never happened before, cause this is critical part and would break tests.

Reply:
Yes, you are right. The type hint was added recently.